src/components/literatureReview/LiteratureReview.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './LiteratureReview.css';
+
+class LiteratureReview extends React.Component{
```

Comment:
A case of the unneeded class-based component. Despite showing their age, class-based components are kept alive by devs who refuse to use functional components when the situation calls for a functional component :P

Just a suggestion, I'm good either way :)